Listenbrainz/repository/FeedRepositoryImpl.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FeedRepositoryImpl.swift
+//  Listenbrainz
+//
+//  Created by Gaurav Bhardwaj on 30/08/23.
+//
+
+import Foundation
+import Alamofire
+import Combine
+
+class FeedRepositoryImpl: FeedRepository {
+    private let authToken = "bd7a7694-d600-406c-befb-13c6f3e564c7"
```

Comment:
get the auth token from settings and don't hardcode it

Reply:
done